doc/src/fix_langevin.txt

```diff
@@ -217,6 +217,10 @@ the particles.  As described below, this energy can then be printed
 out or added to the potential energy of the system to monitor energy
```

Comment:
What about modifying the syntax summary:

  {gjf} value = {vfull} or {vhalf}
    {vfull} = ...
    {vhalf} = ...

Reply:
Hello @athomps, thank you for catching that.